.github/workflows/ci.yml

```diff
@@ -148,7 +173,20 @@ jobs:
     #     setup-python-dependencies: false
 
     - name: Build wheel (Linux)
```

Comment:
Since this step is now identical to the one below for Linux ARM64 I think it makes sense to merge them by removing this one and renaming the below one to just Linux. The only new thing is to run QEMU before that. Same for the tests.

Reply:
I've fixed it